Tools/GNUMake/comps/nvcc.mak

```diff
@@ -112,6 +112,12 @@ ifeq ($(CUDA_VERBOSE),TRUE)
   NVCC_FLAGS += --ptxas-options=-v
 endif
 
+ifeq ($(USE_CUPTI),TRUE)
+  INCLUDE_LOCATIONS += $(SYSTEM_CUDA_PATH)/extras/CUPTI/include
+  LIBRARY_LOCATIONS += ${SYSTEM_CUDA_PATH}/extras/CUPTI/lib64 
+  LIBRARIES += -Wl,-rpath,${SYSTEM_CUDA_PATH}/extras/CUPTI/lib64 -lcupti
```

Comment:
Can we change this to `$(MAKE_CUDA_PATH)`?

Reply:
Done.